src/components/LoginForm/LoginForm.tsx

```diff
@@ -11,17 +11,22 @@ import InputLabel from '@mui/material/InputLabel';
 import OutlinedInput from '@mui/material/OutlinedInput';
 import Stack from '@mui/material/Stack';
 import TextField from '@mui/material/TextField';
-import PropTypes from 'prop-types';
 import { useContext, useState } from 'react';
 import { useForm } from 'react-hook-form';
+import LoginContext from 'src/contexts/LoginContext';
+import { loginSchema } from 'src/schemas/authSchemas';
+import { RESET_PASSWORD, SIGN_IN_URL } from 'src/URLs';
 
 import styles from '/src/components/LoginForm/LoginForm.module.css';
-import LoginContext from '/src/contexts/LoginContext';
-import { loginSchema } from '/src/schemas/authSchemas';
-import { RESET_PASSWORD, SIGN_IN_URL } from '/src/URLs';
-import { resetPassword, signInSignUp } from '/src/utils/auth';
 
-function LoginForm({ changeStateFn }) {
+import LoginFormFields, { OnSubmitProps } from '../../schemas/loginRegisterFormSchemas';
+import { resetPassword, signInSignUp } from '../../utils/auth';
+
+type LoginFormProps = {
+  changeStateFn: () => void;
```

Comment:
I think this should be `changeStateFn: (error: string) => void;` or `changeStateFn: (message: string) => void;`. And again, please consider changing the name to something more specific.